sapi/fpm/fpm/fpm_log.c

```diff
@@ -19,11 +19,7 @@
 #include "fastcgi.h"
 #include "zlog.h"
 
-#ifdef MAX_LINE_LENGTH
-# define FPM_LOG_BUFFER MAX_LINE_LENGTH
-#else
 # define FPM_LOG_BUFFER 1024
```

Comment:
Nit: space should be removed if the nesting is removed.

Reply:
Done